tests/components/RangeInput.stories.tsx

```diff
@@ -1,78 +1,53 @@
+import { FilterGroupContext } from '../../src/components/Filters/FilterGroupContext';
 import { userEvent, within } from '@storybook/testing-library';
-import { SelectableFieldValueFilter } from '../../src/models/SelectableFieldValueFilter';
+import { filterContextValue, filterContextValueDisabled, filterGroupContextValue } from '../__fixtures__/data/filtercontext';
 
 const meta: ComponentMeta<typeof RangeInput> = {
   title: 'RangeInput',
   component: RangeInput,
-};
-
-export default meta;
-
-const selectableFilter: SelectableFieldValueFilter = {
-  selected: true,
-  fieldId: '123',
-  matcher: Matcher.Equals,
-  value: 'test'
-};
-
-const filterContextValue: FiltersContextType = {
-  selectFilter: () => null,
-  applyFilters: () => null,
-  filters: []
-};
-
-const filterContextValueDisabled: FiltersContextType = {
-  selectFilter: () => null,
-  applyFilters: () => null,
-  filters: [selectableFilter]
-};
-
-const filterGroupContextValue: FilterGroupContextType = {
-  searchValue: '',
-  fieldId: '123',
-  setSearchValue: () => null,
-  getCollapseProps: null,
-  getToggleProps: null,
-  isExpanded: null,
-  isOptionsDisabled: null,
-  setIsOptionsDisabled: () => null
-};
-
-export const Primary = (args: RangeInputProps) => {
-  return (
+  argTypes: {
+    inputPrefix: {
+      control: false
+    }
+  },
+  decorators: [(Story) => (
     <SearchHeadlessContext.Provider value={generateMockedHeadless()}>
       <FilterGroupContext.Provider value={filterGroupContextValue}>
-        <FiltersContext.Provider value={filterContextValue}>
-          <RangeInput {...args}/>
-        </FiltersContext.Provider>
+        <Story />
       </FilterGroupContext.Provider>
     </SearchHeadlessContext.Provider>
+  )]
+};
+export default meta;
+
+export const Primary: Story<RangeInputProps> = (args) => {
+  return (
+    <FiltersContext.Provider value={filterContextValue}>
+      <RangeInput {...args}/>
+    </FiltersContext.Provider>
   );
 };
 
-export const Disabled = (args: RangeInputProps) => {
+export const Disabled: Story<RangeInputProps> = (args) => {
   return (
-    <SearchHeadlessContext.Provider value={generateMockedHeadless()}>
-      <FilterGroupContext.Provider value={filterGroupContextValue}>
-        <FiltersContext.Provider value={filterContextValueDisabled}>
-          <RangeInput {...args}/>
-        </FiltersContext.Provider>
-      </FilterGroupContext.Provider>
-    </SearchHeadlessContext.Provider>
+    <FiltersContext.Provider value={filterContextValueDisabled}>
+      <RangeInput {...args}/>
+    </FiltersContext.Provider>
   );
 };
 
 export const DisabledForceDisplayTooltip = Disabled.bind({});
 DisabledForceDisplayTooltip.play = ({ canvasElement }) => {
   const canvas = within(canvasElement);
   const tooltip = canvas.getByText('Unselect an option to enter in a range.').parentElement;
-  tooltip.style.visibility = 'visible';
+  if (tooltip) {
+    tooltip.style.visibility = 'visible';
+  }
```

Comment:
can you check if we actually need the `parentElement` to be visible or just the element return from `canvas.getByText('Unselect an option to enter in a range.')`? If it's the latter, we can remove the if condition.